adrs/20241029-versioning.md

```diff
@@ -2,51 +2,78 @@
 
 **authors:** [Ludo](https://github.com/ludoo),  [Julio](https://github.com/jccb), [Simone](https://github.com/sruffilli) \
 **date:** Oct 29, 2024
+**last update**: Oct 30, 2024
 
 ## Status
 
-Proposed
+Piloting
 
 ## Context
 
-Our current versioning scheme releases new versions based on changes across all modules. This approach was suitable when modules were the primary focus of development. However, with the increasing importance of FAST, this process no longer aligns with our priorities. We need a versioning scheme that reflects the significance of FAST releases and allows for more frequent updates to modules and documentation. The current release process wasn't designed with FAST in mind, causing friction and delaying releases.
+Our current versioning scheme releases new versions based on changes across modules. This approach was suitable when modules were the primary focus of development. However, with the increasing importance of FAST, this process no longer aligns with our priorities. We need a versioning scheme that reflects the significance of FAST releases and allows for more frequent updates to modules and documentation. The current release process wasn't designed with FAST in mind, causing friction and delaying releases.
 
 ## Proposal
 
-Tie major version releases to FAST releases containing breaking changes or new architectural paradigms. Minor version releases will be used for module updates and documentation changes.
+Change the versioning schema as follows:
+
+- **Major Release (X.0.0):** A major release is reserved for changes that introduce breaking changes to the core functionality of the Terraform modules.  This means any modification that requires users to change variables or manipulate state to maintain compatibility. Removing functionality from FAST is also considered a breaking change.
+- **minor release (1.Y.0):** A minor release signifies breaking changes within individual modules or components of the project, while maintaining backward compatibility with the overall structure and purpose of the module collection.
```

Comment:
this is also a bit fuzzy: any breaking change to FAST is a no go and needs to go via a major, breaking changes to modules depend on what they impact